Models.py

```diff
@@ -335,14 +407,15 @@ def getInfoToday(self, period: int=14) -> List[float]:
 
     def predict(self, info: Optional[np.array] = None) -> np.array:
         """
-        Wraps the models predict func
-        
-        Since it gets last 60 days, it gives the predictions
-        for the last 60 days back.  Get the last one to get the applicable day
+        This method will wrap the models predict method. It gives the predictions based on data
+        from the last 60 days back.  Get the last one to get the applicable day.
```

Comment:
This method wraps the model's predict method. It gives the predictions based on data
from the last `num_days` back.  Returns the last one to get the applicable day.